regulation/validation.py

```diff
@@ -560,7 +578,7 @@ def marker_to_target(marker_string):
                 answer = raw_input('Save? y/n: ')
             if answer == 'y':
                 with open(regulation_file, 'w') as f:
-                    f.write(etree.tostring(tree, pretty_print=True))
+                    f.write(etree.tostring(tree, pretty_print=True, encoding='UTF-8'))
```

Comment:
Curious, what's the motivation for this change within this PR?

Reply:
This just makes it so that various Unicode entities like the section sign get printed in their raw form rather than as codes.